moda-contracts/src/ModaRegistry.sol

```diff
@@ -95,10 +113,14 @@ contract ModaRegistry is IModaRegistry, AccessControlEnumerable {
      */
     function addManagers(address[] calldata managers) external {
         for (uint256 i = 0; i < managers.length; i++) {
-            if (_managers[msg.sender].contains(managers[i])) revert DuplicateAddress();
-            if (managers[i] == address(0)) revert AddressCannotBeZero();
-            _managers[msg.sender].add(managers[i]);
-            emit ManagerAdded(msg.sender, managers[i]);
+            if (_managers[msg.sender].contains(managers[i])) {
```

Comment:
I don't think we need this check. managers is an `EnumberableSet`. Sets should be unique. I may be wrong about this type, but I think this is the case. We can prove it with a test

Reply:
I tested passing in the same address twice and found it was added once, so yep removed.